src/client/app/services/exportData.js

```diff
@@ -0,0 +1,48 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import moment from 'moment';
+
+/**
+ * Function to converts the compressed meter data into a CSV formatted string.
+ * @param items The compressed meter data.
+ * @returns output A string containing the CSV formatted compressed meter data.
+ */
+
+function convertToCSV(items) {
+	let csvOutput = 'id:,readings:,timestamp:\n';
+	items.forEach(set => {
+		const data = set.exportVals;
+		const id = set.id;
+		data.forEach(reading => {
+			const info = reading.y;
+			const timeStamp = moment(reading.x).format('dddd MMM DD YYYY hh:mm a');
+			csvOutput += `${id},${info} kwh, ${timeStamp} \n`;
```

Comment:
In addition to the meter ID, I would add the name of the meter. As meter IDs are internal to our system, someone would have no idea what the meter ID represented.